src/test/java/JsonParserTests.java

```diff
@@ -0,0 +1,76 @@
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JsonParserTests {
+
+    Parser parser;
+    Cart comparedWriteCart;
+    Cart comparedReadCart;
```

Comment:
Please do not forget about access modifiers of fields. In this case all fields should be private